backend/app/Listeners/SendDisplayCreatedNoDeviceNotification.php

```
@@ -0,0 +1,23 @@
+class SendDisplayCreatedNoDeviceNotification
+{
+    public function handle(DisplayCreatedNoDevice $event): void
+    {
+        $webhookUrl = config('settings.display_created_no_device_webhook_url');
+        if (!$webhookUrl) {
+            return;
+        }
+
+        Http::post($webhookUrl, [
+            'event' => 'display_created_no_device',
+            'user' => UserWebhookData::from($event->user),
```

PHPStan (2.1.54)

[error] Call to an undefined static method App\Data\UserWebhookData::from().


(staticMethod.notFound)